pkg/pprof/merge.go

```diff
@@ -36,7 +36,7 @@ func (m *ProfileMerge) merge(p *profilev1.Profile, clone bool) error {
 	if p == nil || len(p.StringTable) < 2 {
 		return nil
 	}
-	ConvertIDsToIndices(p)
+	sanitizeReferences(p)
```

Comment:
The only reason we do it here is that `Merge` might be called without `Normalize`. It is a fairly expensive operation, but it's better than a failure